### PR TITLE
Add missing FR translations

### PR DESCRIPTION
**Pull request by stphnpt**
Used in backend for widgets

### CODE REVIEW

**Comment by core23** [comment]
Please squash your commits

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 1]
```diff
@@ -214,6 +214,42 @@
...
+                <source>filter.label_enabled</source>
+                <target>Activé</target>
+            </trans-unit>
+            <trans-unit id="54" resname="filter.label_name">
+                <source>filter.label_name</source>
+                <target>Nom</target>
+            </trans-unit>
+            <trans-unit id="55" resname="filter.label_edited">
+                <source>filter.label_edited</source>
+                <target>Edité</target>
```
Comment (greg0ire) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: "Modifié" would sound a bit more normal I think.

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 2]
```diff
@@ -214,6 +214,42 @@
...
+                <source>show.label_enabled</source>
+                <target>Activé</target>
+            </trans-unit>
+            <trans-unit id="60" resname="show.label_name">
+                <source>show.label_name</source>
+                <target>Nom</target>
+            </trans-unit>
+            <trans-unit id="61" resname="show.label_edited">
+                <source>show.label_edited</source>
+                <target>Edité</target>
```
Comment (greg0ire) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: Same here

**Comment by stphnpt** [comment]
I just copied what already exists in SonataPageBundle ...

**Comment by stphnpt** [comment]
@core23 C

**Comment by stphnpt** [comment]
@core23 Can you squash commits for me ? ... Still learning Git and I don't know how to do that ...

**Comment by greg0ire** [comment]
> I just copied what already exists in SonataPageBundle ...

Oh, that's too bad… let's be consistent I guess…

**Pull request merged**